app/controllers/v0/sessions_controller.rb

```diff
@@ -35,7 +35,13 @@ def saml_callback
 
     def persist_session_and_user!
       @session = Session.new(user_attributes.slice(:uuid))
-      @current_user = User.find(@session.uuid) || Decorators::MviUserDecorator.new(User.new(user_attributes)).create
+
+      if (user_attributes['level_of_assurance'] == 'loa1')
```

Comment:
@austin-martinez dont mind me editing your comment, just added the syntax highlighting :-)